static/app/components/events/eventReplay/staticReplayPreview.tsx

```diff
@@ -91,7 +91,7 @@ const PlayerContainer = styled(FluidHeight)`
   position: relative;
   background: ${p => p.theme.background};
   gap: ${space(1)};
-  max-height: 448px;
+  max-height: 496px;
```

Comment:
noticing a lot of re-used numbers here - e.g. `496` and `480` would` it be useful to make these constants?

Reply:
@michellewzhang I can do that, any ideas where these constants should go?

Reply:
you could probably make a `utils.tsx` file within `eventReplay/eventReplaySection.tsx`!